ocs_ci/ocs/constants.py

```diff
@@ -1244,3 +1248,20 @@
 VOLUMESNAPSHOT = "volumesnapshot"
 
 PERF_IMAGE = "quay.io/ocsci/perf:latest"
+
+ROOK_CEPH_CONFIG_VALUES = [
+    "[global]",
+    "mon_osd_full_ratio = .85",
+    "mon_osd_backfillfull_ratio = .8",
+    "mon_osd_nearfull_ratio = .75",
+    "mon_max_pg_per_osd = 600",
+    "[osd]",
+    "osd_memory_target_cgroup_limit_ratio = 0.5",
+]
+CEPH_DEBUG_CONFIG_VALUES = [
+    "[mon]",
+    "debug_mon = 20",
+    "debug_ms = 1",
+    "debug_paxos = 20",
+    "debug_crush = 20",
+]
```

Comment:
This should be stored as a single string.

Reply:
same

Reply:
fixed now